api/notd/store/saver.py

```diff
@@ -51,3 +56,46 @@ async def create_token_transfer(self, retrievedTokenTransfer: RetrievedTokenTran
             blockHash=retrievedTokenTransfer.blockHash,
             blockDate=retrievedTokenTransfer.blockDate,
         )
+
+    async def create_token_metadata(self, tokenId: int, registryAddress: str, metadataUrl: str, imageUrl: int, name: str, description: str, attributes: str) -> TokenMetadata:
```

Comment:
same here - many of these are not optional but are nullable in the table definition below. please check everything is consistent here.

Reply:
Should I equate to _EMPTY_STRING